docs/team/zawedcvg.md

```diff
@@ -13,14 +13,27 @@ created with JavaFX. It is written in Java, and has about 10 kLoC.
     - Feature: Groups
       - aid in the implementation of an extra parameter "group"
       - add testing for the feature group
+    - Feature: setDefaultCommand
+      - aid in the implementation of the setDefaultCommand
+      - add storage for the modules and the defaultGroups
+    - Bug: Editing groups
+      - fixed a bug where editing groups was throwing errors
+
+
 - **Contribution to the User Guide:**
     - Fix user guide to account for the new parameter group
+    - Add use case for the add command where group is optional
```

Comment:
Hi Neel, where is this use case that you have created? Also maybe you should revisit the terminology since Use Cases exist in Developer Guides and not User Guides.

Reply:
I was implying the part where we can set the group optional in the UG. what do you suppose i can replace the word use cases with?

Reply:
Maybe you can name it an extension to the `add` command as opposed to use case. Use cases imply something very different.